anta/models.py

```diff
@@ -657,6 +679,30 @@ async def wrapper(
 
         return wrapper
 
+    def _handle_failed_commands(self) -> None:
+        """Handle failed commands inside a test.
+
+        There can be 3 types:
+        * unsupported on hardware commands which set the test status to 'skipped'
+        * known EOS error which set the test status to 'failure'
+        * unknown failure which set the test status to 'error'
+        """
+        cmds = self.failed_commands
+        unsupported_commands = [f"'{c.command}' is not supported on {self.device.hw_model}" for c in cmds if not c.supported]
+        if unsupported_commands:
+            msg = f"Test {self.name} has been skipped because it is not supported on {self.device.hw_model}: {GITHUB_SUGGESTION}"
+            self.logger.warning(msg)
+            self.result.is_skipped("\n".join(unsupported_commands))
+            return
+        returned_known_eos_error = [f"'{c.command}' failed on {self.device.name}: {', '.join(c.errors)}" for c in cmds if c.returned_known_eos_error]
+        if returned_known_eos_error:
+            msg = f"Test {self.name} has failed."
+            self.logger.warning(msg)
```

Comment:
`WARNING  [anta.tests.routing.isis.VerifyISISNeighborState] Test VerifyISISNeighborState has failed.`

The test will be marked as failed in the report so here users might ask themselves "Why am I getting a log for this failed test but not the others?".

I think we should either remove this log or provide more context/details just like we do for `unsupported_commands` which provides more context why it was skipped than the actual message in the report.

Reply:
I vote no log - the test has failed and that's enough